### PR TITLE
Added Aave

### PR DESCRIPTION
**Pull request by mviswanathsai**
Added explanations and questions for Aave

### CODE REVIEW

File: src/questions/aave.yaml [hunk 1]
```diff
@@ -0,0 +1,243 @@
...
+  hint: NoHint
+  explanation: The value of an aToken is pegged to the value of the corresponding asset at a 1:1 ratio, and the user accumulates aTokens in their wallet according to the interest rate offered.
+  answerKeys:
+    - A
+  difficultyLevel: "Medium"
+  choices:
+    - content: The number of aTokens in the user's possession keeps increasing while the rate of exchange between atokens and the underlying tokens remains the same
+      key: A
+      
+    - content: The number of aTokens in the user's possession keeps remains the same while the rate of exchange between atokens and the underlying tokens keeps increasing
```
Comment (RobinNagpal) [inline src/questions/aave.yaml]:
```suggestion
    - content: The number of aTokens in the user's possession remains the same while the rate of exchange between atokens and the underlying tokens keeps increasing
```

File: src/questions/aave.yaml [hunk 2]
```diff
@@ -0,0 +1,243 @@
...
+      key: B
+      
+    - content: The number of aTokens in the user's possession keeps increasing while the rate of exchange between atokens and the underlying tokens also increases
```
Comment (RobinNagpal) [inline src/questions/aave.yaml:16]:
```suggestion
    - content: Aave keeps sending part of the supplied tokens as interest every second.
```

File: src/questions/aave.yaml [hunk 3]
```diff
@@ -0,0 +1,243 @@
...
+  explanation: Aave uses debt tokenization to keep count of user's debt called Debt tokens. Debt tokens are interest-accruing tokens that are minted and burned on borrow and repay, representing the debt owed by the token holder and are non-transferrable. 
+  answerKeys:
+    - A
+  difficultyLevel: "Medium"
+  choices:
+    - content: Debt tokenization
+      key: A
+      
+    - content: Calculation and accounting of debt in smart contracts
+      key: B
```
Comment (RobinNagpal) [inline src/questions/aave.yaml:35]: `B` option is also right. Please write some other option.

File: src/questions/aave.yaml [hunk 4]
```diff
@@ -0,0 +1,243 @@
...
+    - content: Both A and B
+      key: D
+      
+
+
+- uuid: 9e2e7ef6-2978-47bc-a6c9-29c38d686436
+  type: SingleChoice
+  content: Debt tokens are transferrable.
+  hint: NoHint
+  explanation: False, debt tokens are not transferrable.
```
Comment (RobinNagpal) [inline src/questions/aave.yaml]: What happens when someone transfers the debt tokens?

File: yarn.lock [hunk 5]
```diff
@@ -1,47 +1,74 @@
-# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (RobinNagpal) [inline yarn.lock]: Why do we need to modify this file? Can you please install `1.22.17`

**Review by RobinNagpal — CHANGES_REQUESTED** [review]

File: src/questions/aave.yaml (hunk 4 above)
Comment (mviswanathsai) [reply src/questions/aave.yaml]: Although debt tokens are modelled on the ERC20/EIP20 standard, they are non-transferrable. Therefore they do not implement any of the standard ERC20/EIP20 functions relating to transfer() and allowance().

**Pull request merged**